jwala-webservices/src/main/java/com/cerner/jwala/ws/rest/v1/service/app/impl/ApplicationServiceRestImpl.java

```diff
@@ -169,6 +173,7 @@ public Response updateResourceTemplate(final String appName,
     public Response deployConf(final String appName, final String groupName, final String jvmName,
                                final String resourceTemplateName, final AuthenticatedUser authUser) {
         LOGGER.info("Deploying the application conf file {} for app {} to JVM {} in group {} by ", resourceTemplateName, appName, jvmName, groupName, authUser.getUser().getId());
+        historyFacadeService.write("", groupService.getGroup(groupName), "Deploying application resource " + resourceTemplateName + " via resources tab", EventType.USER_ACTION_INFO, authUser.getUser().getId());
```

Comment:
We shouldn't be passing an empty string for the hostname parameter. It looks like this needs to be moved into the service layer so the hostname is accessible for each individual deployment.